sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def nl_func(nl, g_stim):
</pre><p>I think something is slightly off with binning g, which is why your answer is slightly lower than what it should be. I wont really be able to tell without doing some numerical debugging, but for a small issue and my guess it is this one...</p><p> </p><p>There is a much easier way to convert into bins though:</p><p>gbin = np.floor((g+6)/0.2)  (I think, so something very close to this -- can check) </p><p>Note this does all of the g at once, instead of one point at a time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I will try this instead of my kludgy nl_func() function.</p>